keylime/cloud_verifier_tornado.py

```diff
@@ -43,6 +46,13 @@
     logger.error("Error creating SQL engine or session: %s", err)
     sys.exit(1)
 
+@event.listens_for(Engine, "connect")
```

Comment:
Is this conditional on the engine being SQLite?

Reply:
The event listener itself isn't, but the `PRAGMA foreign_keys=on` execution doesn't happen unless the database is SQLite. See here: https://github.com/keylime/keylime/pull/1088/files#diff-67033196f7dec303b3ea55793c6c13fb4b03a72a5515239475a871c3d638dfe4R51